lib/webauth.js

```diff
@@ -140,15 +196,15 @@ module.exports = function(authManager) {
       handler.addProxyAuthorizations(req1, req2Options, authPluginSession);     
     },
     
-    getStatus(req, res) {
+    getStatus: Promise.coroutine(function*(req, res) {
       const handlers = authManager.getAllHandlers();
       const result = new StatusResponse();
       for (const handler of handlers) {
         const pluginID = handler.pluginID;
         const authPluginSession = util.getOrInit(req.session, pluginID, {});
         let status;
         try {
-          status = handler.getStatus(authPluginSession); 
+         status = handler.getStatus(authPluginSession); 
```

Comment:
ultra nitpicking mode on:  this space shouldn't have been removed :)

Reply:
OK, I'll update.